biz.aQute.resolve/src/biz/aQute/resolve/Bndrun.java

```diff
@@ -161,6 +165,23 @@ public int compare(VersionedClause a, VersionedClause b) {
 	}
 
 	/**
+	 * Copy the calculated <code>-runbundles</code> into a directory.
+	 *
+	 * @param targetDir the directory in which <code>-runbundles</code> are
+	 *            placed
+	 * @throws Exception
+	 */
+	public void runbundles(File targetDir) throws Exception {
```

Comment:
Why don't you just call `Project.exportRunbundles(null, targetDir)` from  the mojo? This a duplicate of that method. You have made some improvements here using `Files`. Better to fix the original method. 

Note https://github.com/bndtools/bnd/issues/1625 which these methods suffer from. So better to have one method we can fix once.